docs/lazy-adr/adr-005-decouple-blockid-and-partsetheader.md

```diff
@@ -0,0 +1,49 @@
+# ADR 005: Decouple the `PartSetHeader` from the `BlockID`
+
+## Changelog
+
+- 2021-08-01: Initial Draft
+
+## Context
+
+Celestia has multiple commits to the block data via the `DataHash` and the `PartSetHeader` in the `BlockID`. As stated in the #184, we no longer need the `PartSetHeader` for this additional commitment to the block’s data. However, in the short-medium term, we are still planning on using the `PartSetHeader` to be used for block propagation during consensus. This means that we will remove the `PartSetHeader` from as many places as possible, but keep it in the `Proposal` struct.
```

Comment:
1. Link to #184
2. This might be a nit, but can you use dumb quotes `'` instead of smart quotes `’` throughout the document?

Reply:
nice catch, link updated here https://github.com/celestiaorg/lazyledger-core/pull/482/commits/8421c27559531d6ae31bee4d58eb4a771d7a1bc2

and the switched to using normal quotes https://github.com/celestiaorg/lazyledger-core/pull/482/commits/103f53e71fcfeb7171d87069aac8da660044d2bd

Reply:
Oh no, it looks like 103f53e changed backticks to `'`, so all the code blocks are now single-quoted. Can you change those back to backticks?

Reply:
Actually, it looks like 103f53e _only_ changed backticks to single quotes, but not smart quotes to dumb quotes. (This is probably due to smart quotes being hard to search for in some editors, hence the whole reason to not use them!)

Reply:
@adlerjohn next time better use the suggestion-tool for comments like this.